packages/apps/graph-client/src/pages/transactions/[key].tsx

```diff
@@ -116,7 +116,7 @@ const RequestKey: React.FC = () => {
                     transaction.result.goodResult && (
                       <Notification
                         intent="positive"
-                        icon={<SystemIcon.Check />}
+                        icon={<MonoCheck />}
```

Comment:
![image](https://github.com/kadena-community/kadena.js/assets/112877112/0ae59447-327a-4a87-bcf9-931ad6e09245)

Same here

Reply:
btw fixing this in the notification v2